### PR TITLE
test(cmp): remove redundant test

### PR DESCRIPTION
**Pull request by FlyCloudC**
Removed tests in `cmp_test.mbt` which is
1. Duplicate of doc tests in `cmp.mbt`
2. Unnecessary, since maximum and minimum are generic functions

### CODE REVIEW

File: cmp/cmp_test.mbt [hunk 1]
```diff
@@ -13,108 +13,17 @@
...
-  assert_eq(@cmp.minimum(-3, -5), -5)
-
-  // Edge case with zero
-  assert_eq(@cmp.maximum(0, -1), 0)
-  assert_eq(@cmp.minimum(0, -1), -1)
+test "minimum return first when equal" {
+  let fst = []
+  let snd = []
+  minimum(fst, snd).push(0)
+  inspect(fst, content="[0]")
```
Comment (peter-jerry-ye) [inline cmp/cmp_test.mbt]: These might be worth adding to the doc comment
